crates/turbopack-node/src/node_rendered_source.rs

```diff
@@ -89,7 +89,7 @@ impl NodeRenderContentSource {
     /// Checks if a path matches the regular expression
     async fn is_matching_path(&self, path: &str) -> Result<bool> {
         // TODO(alexkirsz) This should probably not happen here.
-        if path.starts_with('_') {
```

Comment:
Don't know exactly what this is for 🤔

had to change it for the data route

Reply:
This was to ensure that `_next/...` routes would not be caught by this content source. We had an issue where some catch-all Next.js routes, e.g. `[[...slug]]`, would also catch `_next/...` routes.

Reply:
IIRC @sokra fixed this somewhere else so this should no longer be necessary.